core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -2106,32 +2106,65 @@ object TestUtils extends Logging {
       s"There still are ongoing reassignments", pause = pause)
   }
 
-  def addAndVerifyAcls(broker: KafkaBroker, acls: Set[AccessControlEntry], resource: ResourcePattern): Unit = {
-    val authorizer = broker.dataPlaneRequestProcessor.authorizer.get
+  /**
+   * Find an Authorizer that we can call createAcls or deleteAcls on.
+   */
+  def pickAuthorizerForWrite[B <: KafkaBroker](
```

Comment:
It's possible, but I think it would be a lot of extra work to convert the test to do that. There are some other tests that do go through the admin API (citation needed?) so hopefully we can just rely on those